Jarvis-Platform/jarvis-studio
# Hotfix/notification drawer toggle

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Fix Notification drawer toggle


## Todos
- [x] Fix click action


## Deploy Notes
Review, test & merge.


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout hotfix/notification-drawer-toggle
```

1. Click on notification icon in header => Notification drawer should be on screen
2. Click outside of the notification drawer => It should be closed


## Impacted Areas in Application
- `App.vue`
- `src/components/app/NotificationContent.vue`
